packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.md

```diff
@@ -129,6 +129,14 @@ a ?? (b && c) ?? d;
 a ?? (b && c && d);
 ```
 
+### `ignorePrimitives`
```

Comment:
There's a [`lint:markdown` complaint](https://github.com/typescript-eslint/typescript-eslint/actions/runs/4210554347/jobs/7308326641) blocking the PR build.